src/client/app/helpers/helpers.d.ts

```diff
@@ -0,0 +1,2 @@
+declare let DD_SERVICE: string
+declare let DD_ENV: string
```

Comment:
Related to previous comments as well. This won't be necessary if we use `process.env`.

Reply:
This will be taken offline as well

Reply:
updates: worth trying to get this to work by itself without the help of the `eslintrc` config file. maybe this declaration file needs to be directly imported or something? 🤔